controllers/policy_utils.go

```diff
@@ -144,6 +144,18 @@ func reconcilePolicy(ctx context.Context, client client.Client, reconciler admis
 
 	return ctrl.Result{}, nil
 }
+func setPolicyAsActive(policy v1alpha2.Policy) {
+	policy.SetStatus(v1alpha2.PolicyStatusActive)
+	apimeta.SetStatusCondition(
+		&policy.GetStatus().Conditions,
+		metav1.Condition{
+			Type:    string(v1alpha2.PolicyActive),
+			Status:  metav1.ConditionTrue,
+			Reason:  "PolicyActive",
+			Message: "The policy webhooks are created",
+		},
+	)
```

Comment:
This is good. I would also set this same condition to `ConditionFalse` at the beginning of the reconciliation loop.

This means that if we early exit because of an error, the condition on the policy is updated always, and it's set to `False`.